CRM/Sqltasks/Task.php

```diff
@@ -405,6 +410,9 @@ public function execute($params = []) {
       // run action
       try {
         $action->execute();
+        if (get_class($action) == "CRM_Sqltasks_Action_ReturnValue") {
+          self::$return_values[$action->return_key] = $action->return_value;
```

Comment:
To help users catch errors, let's add a warning to the task log if we set a value that is `!empty()` in `self::$return_values` (i.e. if we overwrite existing parameters).